### DISCUSSION TITLE
Fermata Mirror and Fermata FS Mirror

### DISCUSSION THREAD

**Discussion by fwon2638** [AndreyPavlenko/Fermata#372] [Q&A]
How to use these new functions?

**Comment by AndreyPavlenko** [comment]
Before the first start, start Fermata Auto on your phone, connect to AA, start the Mirror app and grant the permissions on your phone.

**Reply by fwon2638** [reply]
Is the Mirror app a separate app?  Can't find it.

**Reply by AndreyPavlenko** [reply]
You should have 3 icons in the AA launcher - Fermata Auto, Fermata Mirror and Fermata FS Mirror.

**Reply by giusecs** [reply]
hi, sorry but I don't see the two new icons either. Is AA an app to install? Which?

**Reply by AndreyPavlenko** [reply]
For Android 14 you need a head unit emulator.

**Reply by AndreyPavlenko** [reply]
https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1916776751

**Reply by lopropio** [reply]
**Updated to 1.9.9:** Hi Andrey, when i follow the steps described above, i become the following message on the phone:
`Unlock the phone and grant the requested authorizations`
But the phone is neither locked nor is the request for grant the permissions displayed. If I use the icon "Fermata Mirror" i become a blach window with 2 tiny icons "home" and "arrow for back".
Other questions, which icon is the right one to start Mirror app? And what is the difference between Fermata Mirror and Fermata FS?
Thx in advance

**Reply by AndreyPavlenko** [reply]
Start Fermata Auto on your phone before connecting to AA. FS stands for Full Screen.

**Comment by Edu452** [comment]
Hola soy nuevo y utilizo desde hace tiempo Fermata y es una aplicación que me encanta. Actualice la última versión en mi OPPO X3 Pro y después de varios intentos, conseguí que apareciese en la pantalla del vehículo los dos nuevos iconos, me pedía que diese los permisos pero observé que mi móvil los tenía bloqueados y terminé consiguiendo que se habilitarán.
Mi pregunta es como funcionan estos dos nuevos iconos y que ajustes tengo que realizar para su correcto funcionamiento, gracias de antemano.
![Screenshot_2024-02-14-16-11-13-79_fc704e6b13c4fb26bf5e411f75da84f2](https://github.com/AndreyPavlenko/Fermata/assets/160119379/fc66b927-b801-4c24-8f84-5d3d1c6b3ac3)
![Screenshot_2024-02-14-16-10-26-23_31f2d1fb8a9cb6537600cbf449362b94](https://github.com/AndreyPavlenko/Fermata/assets/160119379/d3fcd2e8-d213-43e0-afc7-939e3f153732)
![Screenshot_2024-02-14-16-08-28-78_fc704e6b13c4fb26bf5e411f75da84f2](https://github.com/AndreyPavlenko/Fermata/assets/160119379/762dc13e-046d-4fbc-861c-6307c880cd3e)
![Screenshot_2024-02-14-16-08-41-52_fc704e6b13c4fb26bf5e411f75da84f2](https://github.com/AndreyPavlenko/Fermata/assets/160119379/a30fc0d4-1fe0-450b-ae37-19b899213156)

**Comment by embiot78** [comment]
Hello. Fermata Mirror and Fermata  FS  Mirror don't appear on  real headunits (renault kaptur  & chery tiggo8PM), but they appears on DHU emulator.  What is the reason? 
My phone is Realme GT2Pro (android 14)

![photo1709462735](https://github.com/AndreyPavlenko/Fermata/assets/162015740/59e07230-353e-4985-ae3e-a8d879d04f20)
![WhatsApp Image 2024-03-03 at 11 57 49](https://github.com/AndreyPavlenko/Fermata/assets/162015740/447e079f-603f-4229-9ce9-dd22aa3e61f4)
![WhatsApp Image 2024-03-03 at 12 41 58](https://github.com/AndreyPavlenko/Fermata/assets/162015740/e570f49a-3402-4382-a427-7ccdb6c04c5e)

**Reply by produmann** [reply]
the same situation, there is an application on the emulator , but there are none in the car!? what

**Reply by AndreyPavlenko** [reply]
In the car AA does not allow apps installed not from the google play.

**Comment by SOFO888** [comment]
Hey Andrey, 

Just a question concerning Fermata FS: does an app which is declared as POI app also start as startup app on Android Auto? 

Thank you.

**Reply by AndreyPavlenko** [reply]
I've not tested, but I think it doesn't.

**Comment by Gordian72** [comment]
Io ho aggiornato l'applicazione alla 1.9.9 ma delle 2 nuove icone non c'è nulla. Mi può aiutare qualcuno.

**Reply by Pillo1964** [reply]
Io ho installato fermata xtream su android 16
Usando kinginstaller e shizuku.ora tutto funzione perfettamente.lo provato su più cellulare e funziona.

**Comment by luitruda** [comment]
hello instead they appear in AA to me it doesn't let me do anything it doesn't give me any configuration options what am I doing wrong or is there a guide to help me understand how it works

**Comment by olegdeacon** [comment]
Will this work on android16?
